api/test/shell/cli_test.sh

```diff
@@ -117,47 +117,8 @@ if [[ `grep -c "INFO" ./error.log` -eq '0' ]]; then
     exit 1
 fi
 
-# run on a different path
-workDir=$(pwd)
-rm -rf html
-mkdir html
-cd html
-echo "hi~" >> index.html
-APISIX_API_WORKDIR=$workDir $workDir/manager-api &
-sleep 5
-
-res=$(curl http://127.0.0.1:9000)
-$workDir/manager-api stop
-sleep 6
-cd -
-rm -rf html
-
-if [[ $res != "hi~" ]]; then
-    echo "failed: manager-api cant run on a different path"
-    exit 1
-fi
 clean_up
 
-# run with -p flag out of the default directory
-workDir=$(pwd)
-distDir=/tmp/manager-api
-cp -r $workDir $distDir
-cd $distDir && rm -rf bin && mkdir bin && mv ./manager-api ./bin/
-cd $distDir && rm -rf html && mkdir html && echo "hi~" >> html/index.html
-cd $distDir/bin && ./manager-api -p $distDir &
-sleep 5
-
-res=$(curl http://127.0.0.1:9000)
-$distDir/bin/manager-api stop
-sleep 6
-rm -fr $distDir
-
-if [[ $res != "hi~" ]]; then
-    echo "failed: manager-api can't run with -p flag out of the default directory"
-    exit 1
-fi
-cd $workDir && git checkout conf/conf.yaml
-
 # test start info
```

Comment:
we should add test cases for changes.

Reply:
Yes, I can add tests for when the web build is not embedded, as comparing less content is easy.
But when web build is embedded, I'm not sure how to do it.
Do you have a plan?